### DISCUSSION TITLE
WebUI will not load

### DISCUSSION THREAD

**Discussion by Drudgen44** [qdm12/ddns-updater#443] [Q&A] [answered]
ddns-updater is updating my A record with Cloudflare just fine and there are no errors in the log. However when I go to access the web interface I get this error below. I am running this image with Gluetun network with a VPN if that makes a difference. Since Portainer reserves port 8000 I did a   - "8001:8000/tcp" entry and load it through 8001.  Any ideas?

`unversioned API: requested URI not found`

**Comment by Drudgen44** [comment, marked as answer]
Seems that changing everything internal and external to port 8001 did the trick. The WebUI loads now.

**Reply by qdm12** [reply]
Gluetun listens on port 8000 already, so you were trying to access the gluetun control server (why you get `unversioned API: requested URI not found`).
